i18n/zh-CN/docusaurus-plugin-content-blog/2023-10-25-biweekly-newsletter/index.md

```diff
@@ -1,6 +1,6 @@
 ---
 slug: 2023-10-25-biweekly-newsletter
-title: KCL 社区开源双周报 (2023 10.12 - 10.25) | IDE 插件支持引用查找、包管理集成 ArtifactHub
+title: KCL 社区开源双周报 (2023 10.12 - 10.25) | FindReferences and Rename in IDE, ArtifactHub integration in KPM!
```

Comment:
The Chinese and English titles seem to have been used incorrectly.

Reply:
sorry, I noticed that too and thought I've already fixed that